payment/datastore.go

```diff
@@ -62,6 +71,75 @@ func NewPostgres(databaseURL string, performMigration bool, dbStatsPrefix ...str
 	return nil, err
 }
 
+// CreateKey FIXME
+func (pg *Postgres) CreateKey(merchant string, encryptedSecretKey string, nonce string) (*Key, error) {
+	// interface and create an api key
+	var key Key
+	err := pg.DB.Get(&key, `
+			INSERT INTO api_keys (merchant_id, encrypted_secret_key, nonce)
+			VALUES ($1, $2, $3)
+			RETURNING *
+		`,
+		merchant, encryptedSecretKey, nonce)
+
+	if err != nil {
+		return nil, err
+	}
+	key.SetSecretKey()
+
+	return &key, nil
+}
+
+// DeleteKey FIXME
+func (pg *Postgres) DeleteKey(id uuid.UUID, delaySeconds int) (*Key, error) {
+	// interface and create an api key
+	var key Key
+	err := pg.DB.Get(&key, `
+			UPDATE api_keys
+			SET expiry=(current_timestamp + $2)
+			WHERE id=$1
+			RETURNING *
+		`, id.String(), fmt.Sprintf("%vs", delaySeconds))
+
+	if err != nil {
+		return nil, err
+	}
+
+	if &key == nil {
+		return nil, fmt.Errorf("No rows were affected")
+	}
+	key.SetSecretKey()
+
+	return &key, nil
+}
+
+// GetKeys returns a list of active API keys
+func (pg *Postgres) GetKeys(merchant string, showExpired bool) (*[]Key, error) {
+	expiredQuery := "AND (expiry IS NULL or expiry > CURRENT_TIMESTAMP)"
+	if showExpired {
+		expiredQuery = ""
+	}
+
+	var keys []Key
+	err := pg.DB.Select(&keys, `
+			SELECT * FROM api_keys
+			WHERE merchant_id = $1
+
+		`+expiredQuery,
+		merchant)
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Turn this into a log message?

Reply:
Ended up just removing this :) https://github.com/brave-intl/bat-go/pull/285/commits/f5ca28286a4358dce3ee80d23a3f1c8bdc66e717